document/manuscript.md

```diff
@@ -41,7 +41,7 @@ projects, since they enable the organisation and sharing of
 programming tasks between different remote contributors. Here, we
 introduce the main features of GitHub, a popular web-based platform
 which offers a free and integrated environment for hosting the source
-code, documentation and web page for open source projects. GitHub also
+code, documentation and project-related web content for open source projects. GitHub also
```

Comment:
why not only: "web content"
project-related is a bit redundant given the sentence ends with projects